dlp/api/Snippets/ReidentifyContentUsingDeterministic.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START dlp_reidentify_deterministic]
+
+using System;
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Google.Protobuf;
+
+public class ReidentifyContentUsingDeterministic
+{
+    public static ReidentifyContentResponse ReidentifyDeterministic(
+        string projectId,
+        string text,
+        string keyName,
+        string wrappedKey,
+        InfoType surrogateType = null)
+    {
+        // Instantiate the client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the infotype if null which will be used as surrogate type and infotype both.
+        var infoType = surrogateType ?? new InfoType { Name = "PHONE_TOKEN" };
```

Comment:
Please use the same surrogate in both de-id and re-id.

Reply:
done